Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -1307,7 +1307,11 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
             "enable-accelerated-2d-canvas",
             _("Enable accelerated 2D canvas"),
             _("Whether to enable accelerated 2D canvas"),
+#if USE(SKIA)
             FEATURE_DEFAULT(CanvasUsesAcceleratedDrawing),
+#else
+            FALSE,
+#endif
```

Comment:
This is an existing issue, but this should always be FALSE, since this setting does nothing.

Reply:
Er, OK... but shouldn't we make it an alias of the newer setting?

(We probably should have just undeprecated the original setting rather than add the new one, but whatever.)

Reply:
Will change this to always FALSE before landing.